### PR TITLE
jin - w2 

### PR DESCRIPTION
**Pull request by jiiinCho**
Link to my deployed project: https://technigo-w2-news.netlify.app

### CODE REVIEW

File: code/index.html [hunk 1]
```diff
@@ -1,22 +1,149 @@
 <!DOCTYPE html>
-<html>
+<html lang="en">
   <head>
-    <meta charset="utf-8" />
-    <title>Project 2: The News Site</title>
-    <link rel="stylesheet" href="./style.css" />
-    <!-- dont forget the meta tag for responsiveness -->
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
```
Comment (CamillaHallberg) [inline code/index.html:5]: Nice to make sure of compatibility to IE=edge! 👍🏽

File: code/index.html [hunk 2]
```diff
@@ -1,22 +1,149 @@
...
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>The Book Magazine</title>
+    <link rel="stylesheet" href="style.css" />
+    <!-- google fonts -->
+    <link rel="preconnect" href="https://fonts.googleapis.com" />
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Lato:wght@300;700&family=Libre+Baskerville:wght@700&family=Luxurious+Roman&display=swap"
+      rel="stylesheet"
```
Comment (CamillaHallberg) [inline code/index.html:14]: I like that you made a comment about this being Google Fonts and also nice to see multiple fonts being used.

File: code/index.html [hunk 3]
```diff
@@ -1,22 +1,149 @@
...
+        </p>
+      </div>
     </section>
 
-    <section class="">
+    <section class="popular">
       <!-- The other news  -->
+      <div class="popular-heading">
+        <h4 class="popular-title">Popular articles</h4>
+        <div class="popular-divider"></div>
```
Comment (CamillaHallberg) [inline code/index.html:59]: Nice touch with a divider to your popular-heading!

File: code/style.css [hunk 4]
```diff
@@ -1,4 +1,209 @@
...
+}
+.popular-card {
+  padding: 1em;
+  -webkit-box-shadow: 2px 2px 5px 0px rgba(0, 0, 0, 0.56);
+  -moz-box-shadow: 2px 2px 5px 0px rgba(0, 0, 0, 0.56);
+  box-shadow: 2px 2px 5px 0px rgba(0, 0, 0, 0.56);
+  transition: all 0.25s ease-in;
+}
+.popular-card:hover {
+  transform: scale(1.01);
```
Comment (CamillaHallberg) [inline code/style.css:170]: I really like these effects put to your popular cards. That slow ease in when you mouse over is a really nice, professional looking touch! 😃

**Review by CamillaHallberg — COMMENTED** [review]
Wow Jin! 🌟 What an amazing job you have done with this page. The styling of your page is really pleasing to watch and I really enjoyed the effects of for example when hovering the navbar the links change their color to red and also that small touch of the cursor changing. Also just the entire layout and your usage of colors. It's very clean and informative in a nice way. I really liked how you have combined both Flexbox and Grid and how it's responsive to the different devices. I'm also taking notes on how you managed to make your site responsive without using too many media queries. Your mark-up is easily read and follows a good structure, where you've also made comments which is such a good thing when you're looking at someone else's code. 

All in all, you've done a fantastic job and it was really fun and interesting to review your site! 🥳🌟🙌🏽

File: code/style.css (hunk 4 above)
Comment (jiiinCho) [reply code/style.css:170]: > I really like these effects put to your popular cards. That slow ease in when you mouse over is a really nice, professional looking touch! 😃

I'm using this site for box shadow settings where I can just adjust slides and copy the result code into css!  Here is for your reference :) 
https://cssgenerator.org/box-shadow-css-generator.html

Comment (CamillaHallberg) [reply code/style.css:170]: That's awesome! Bookmarking it straight away, thanks! :)

**Pull request closed without merging**